src/LIBSVM.jl

```diff
@@ -223,6 +228,36 @@ function check_train_input(X, y, kernel)
     end
 end
 
+function data2gram(kernel_function, X)
+    ntrain = size(X, 2)
+    gram = Array{Float64}(undef, ntrain, ntrain)
+    for i=1:ntrain
+        for j in 1:i
```

Comment:
```suggestion
    for i ∈ 1:ntrain, j ∈ 1:i
```

Reply:
I notice that all
```julia
for i in 1:n
for i = 1:n
for i ∈ 1:n
```
are options for iterator assignment in Julia. Are they all functionally and semantically the same? If yes I'd suggest settling on one version and I'd make sure that at least all the for-loops that are modified/added in this PR use that version.

Reply:
yes, all of them are the same. I'm okay with any one. `∈` is just my personal preference.